ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+#TODO testing if missing YQ is a $PATH problem
+export PATH="$PATH:/usr/local/bin"
+
+
+
```

Comment:
What did testing conclude?
```suggestion
export PATH="$PATH:/usr/local/bin"

```

Reply:
It works! Do you think this should live here or we should bake it into the upi image? I have noticed other jobs using a different yq, but they install it in the job to `tmp` and then use a fully qualified `/tmp/yq` so I don't think putting it in the image will break them.

Reply:
Let's do the PATH stuff here for now. But, yes, I think we should adjust the path in the image.